pcsx2/ImGui/ImGuiOverlays.cpp

```diff
@@ -51,6 +51,8 @@
 #include <tuple>
 #include <unordered_map>
 
+float position_y = 0;
```

Comment:
Please don't add additional global variables, polluting the global namespace.

This is also going to break if you turn recording off. Have a local in the parent function, and pass it in by reference.